sshfs/config.py

```diff
@@ -1,29 +1,44 @@
 import getpass
 from contextlib import suppress
 from pathlib import Path
+import asyncssh
 
 from asyncssh.config import SSHClientConfig
 
 SSH_CONFIG = Path("~", ".ssh", "config").expanduser()
 
 
-def parse_config(
-    *, host, user=(), port=(), local_user=None, config_files=None
-):
+def parse_config(*, host, user=(), port=(), local_user=None, config_files=None):
     if config_files is None:
         config_files = [SSH_CONFIG]
 
     if local_user is None:
-        with suppress(KeyError):
+        with suppress(OSError):  # Use OSError as getuser() might raise this.
```

Comment:
Let's keep `KeyError` too. I see that it raises `OSError` now in 3.13, but that is not true for older versions.
```suggestion
        with suppress(OSError):  # Use OSError as getuser() might raise this.
```